src/nxdoc/nuxeo-server/workflow/workflow-how-to-index/how-to-refresh-the-task-widget-on-the-summary-tab.md

```diff
@@ -56,11 +56,10 @@ history:
         version: '1'
 
 ---
-{{! excerpt}}
 
-If you start workflow automatically using the [Workflow > StartWorkflow](http://explorer.nuxeo.org/nuxeo/site/distribution/Nuxeo%20Platform-8.10/viewOperation/Context.StartWorkflow) operation and that your workflow's first node creates a task to the workflow initiator, you need to use in the input chain the **[User Interface > Raise Seam events](http://explorer.nuxeo.com/nuxeo/site/distribution/Nuxeo%20Platform-8.10/viewOperation/Seam.RaiseEvents)** operation, with the value "<span style="color: rgb(34,34,34);">`**workflowNewProcessStarted**`" for the event name.</span>
+If you start workflow automatically using the [Workflow > StartWorkflow](http://explorer.nuxeo.com/nuxeo/site/distribution/Nuxeo%20Platform%20LTS%202016-8.10/viewOperation/Context.StartWorkflow) operation and that your workflow's first node creates a task to the workflow initiator, you need to use in the input chain the **[User Interface > Raise Seam events](http://explorer.nuxeo.com/nuxeo/site/distribution/Nuxeo%20Platform-8.10/viewOperation/Seam.RaiseEvents)** operation, with the value `workflowNewProcessStarted` for the event name.
```

Comment:
```suggestion
If you start workflow automatically using the [Workflow > StartWorkflow](http://explorer.nuxeo.com/nuxeo/site/distribution/8.10/viewOperation/Context.StartWorkflow) operation and that your workflow's first node creates a task to the workflow initiator, you need to use in the input chain the **[User Interface > Raise Seam events](http://explorer.nuxeo.com/nuxeo/site/distribution/8.10/viewOperation/Seam.RaiseEvents)** operation, with the value `workflowNewProcessStarted` for the event name.
```

Reply:
It's easier to put the shorter version of the link to maintain it by doing a Find and Replace when necessary.